io/net/virt-net/network_virtualization.py

```diff
@@ -166,7 +166,7 @@ def setUp(self):
             if result.exit_status:
                 self.fail("failed to enable debug mode")
         else:
-            self.log("failed to enable debug mode")
+            self.log.debug("failed to enable debug mode")
```

Comment:
why not only self.log.info

Reply:
yes.. even the message should change.. as test should continue even with debug disabled

Reply:
Done